collin/index.html

```diff
@@ -0,0 +1,27 @@
+<html lang="en-US">
```

Comment:
```suggestion
<!DOCTYPE html>
<html lang="en-US">
```

Still need that DOCTYPE for valid HTML babyyyyy

Reply:
![oopsie](https://user-images.githubusercontent.com/45298941/102269890-302c3a00-3f15-11eb-8eb6-ec52474198f7.jpg)